src/xFactor/Judge.java

```diff
@@ -0,0 +1,104 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Judge extends Person {
+	private static int maxFavourites;
+	private static int numberOfJudges;
+	private static int countInstances;
+	private static ArrayList<Judge> judges = new ArrayList<Judge>(numberOfJudges);
+	private int vote;

```

Comment:
the vote is not something engaged to the Judge. One judge can have multiple votes for different participants, done like this, you will always override it